docs/integrations/delta.md

```diff
@@ -20,7 +20,16 @@ Because lakeFS is format-agnostic, you can save data in Delta format within a la
 
 {% include toc.html %}
 
-## Beta: Viewing Delta table changes in lakeFS 
+## Beta: Viewing Delta Lake table changes in lakeFS 
+
+### Enable the Delta Lake tables diff
+
+The Delta Lake diff functionality is enabled using a binary plugin. This binary plugin can be found as part of the latest lakeFS
+[release](https://github.com/treeverse/lakeFS/releases/latest) (starting from 0.100.0), and can also be built locally using the `make delta-plugin` command
+(run from lakeFS's repository root). After getting it, place it under `~/.lakefs/plugins/diff`.  
+lakeFS will look for a binary named `delta_diff` under `~/.lakefs/plugins/diff` if not configured otherwise.
```

Comment:
If I use a Docker image, do I get it built-in?  Because that sounds like a _really_ good plan :-)  May be worth pulling out what is needed to use with K8s / Docker.

Reply:
Yes, the docker image is ready to roll

Reply:
Since most of our users presumably use Docker, maybe first mention that this part is only required when using the binary.